docs/edgeanalytics/kafka-options.md

```diff
@@ -0,0 +1,186 @@
+1. Click 'Create'
+
+### Connecting to AMQ Streams
+
+1. In a terminal, run the commands below to get certificates and keystores necessary to connect:
+
+    ```sh
+    # Get Kafka bootstrap route; value will be referred to as <RouteURL> later
+    oc get routes my-cluster-kafka-bootstrap -n amq-streams -o=jsonpath='{.status.ingress[0].host}{"\n"}'
+
+    # Extract server public cert, client public cert, and client private key
+    oc extract secret/my-cluster-cluster-ca-cert -n amq-streams --keys=ca.crt --to=- > ca.crt
+    oc extract secret/my-cluster-client-ca-cert -n amq-streams --keys=ca.crt --to=- > user.crt
+    oc extract secret/my-cluster-client-ca -n amq-streams --keys=ca.key --to=- > user.key
+    ```
+
+1. Use the methods depending on application type:
+
+    * **Python applications**: use the [`streamsx-kafka-make-properties` command](https://ibmstreams.github.io/streamsx.kafka/docs/user/UsingRHAmqStreams/#when-you-have-a-python3-environment) to create a properties file.
+    * **SPL applications**:
+
+        1. Create the truststore and keystore manually:
+
+            ```sh
+            keytool -import -trustcacerts -alias root -file ca.crt -keystore truststore.jks -storepass trustpassword -noprompt
+
+            openssl pkcs12 -export -in user.crt -inkey user.key -name client-alias -out ./keystore.pkcs12 -noiter -nomaciter -passout keypassword
+
+            keytool -importkeystore -deststorepass password -destkeystore ./keystore.jks -srckeystore ./keystore.pkcs12 -srcstoretype pkcs12 -srcstorepass keypassword
+            ```
+
+        1. Populate a `kafka.properties` file with the following values:
+
+            ```properties
+            bootstrap.servers=<RouteURL>
+            security.protocol=SSL
+            ssl.keystore.type=JKS
+            ssl.keystore.password=keypassword
+            ssl.key.password=keypassword
+            ssl.keystore.location={applicationDir}/etc/keystore.jks
+            ssl.endpoint.identification.algorithm=https
+            ssl.truststore.type=JKS
+            ssl.truststore.password=trustpassword
+            ssl.truststore.location={applicationDir}/etc/truststore.jks
+            ```
+
+        1. Copy the JKS files and `kafka.properties` to `etc/` in the SPL application workspace
+
+For more information, see [Using streamsx.kafka with Red Hat AMQ Streams](https://ibmstreams.github.io/streamsx.kafka/docs/user/UsingRHAmqStreams/) documentation.
+
+***
+
+## Event Streams in IBM Cloud
+
+[IBM Event Streams](https://www.ibm.com/cloud/event-streams) builds on top of open source Apache Kafka to offer enterprise-grade event streaming capabilities.
+
+### Provisioning Event Streams
+
+1. Log in to [IBM Cloud](https://cloud.ibm.com) or create an account if you do not have one.
+1. Visit [Event Streams](https://cloud.ibm.com/catalog/services/event-streams) in the catalog.
+1. Select a region (e.g. Dallas, Frankfurt)
+1. Select a plan (e.g. Lite)
```

Comment:
Can you please add a comment that the Lite plan only allows one topic which might not work with some of the samples?